Update README.rst (#229)
ansible-collections/overview: README.rst

**Classic Ansible**

> * Single repository `ansible/ansible <https://github.com/ansible/ansible>`_.

Reviewer: ```suggestion
* Single repository `ansible/ansible <https://github.com/ansible/ansible>`_
```

> * Single package called `ansible`.
> * `ansible` had major releases twice a year.

Reviewer: ```suggestion
* Single package called `ansible`
* `ansible` had major releases twice a year
```

> * New features go into the next major release - ie worst case you need to wait 6 months.

Reviewer: ```suggestion
* New features go into the next major release, i.e., in the worst case, you need to wait 6 months.
```

Now: Ansible 2.10 and later
---------------------------

* The ``ansible/ansible`` (`ansible-base`) repository only contains:

>   * The core Ansible programs, ``ansible-{playbook,galaxy,doc,test,etc}``.
>   * Some documentation.
>   * A tiny subset of modules and plugins to allow for a functioning controller.

Reviewer: ```suggestion
  * The core Ansible programs, ``ansible-{playbook,galaxy,doc,test,etc}``
  * Some documentation
  * A tiny subset of modules and plugins to allow for a functioning controller
```

  * Together this will be known as ``ansible-base``.
* The rest of the modules and plugins have been moved into various "collections"

  * Ansible Collections:

    * Can be released independently of ansible-base and Ansible, at whatever release cycle/cadence the collection maintainer prefers.

>     * Will have their own repo (GitHub, GitLab, etc) with dedicated backlog, ie no more shared massive issue & PR backlog.

Reviewer 2: ```suggestion
    * Will have their own repo (GitHub, GitLab, etc) with dedicated backlog, i.e. no more shared massive issue & PR backlog.
```

Documentation
==============

> * `Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_.

Reviewer 3: ```suggestion
* `Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_.
```
Should we make the other lines in this section contain periods as well?

Documentation
==============

> * `Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_.

Reviewer: None of these should have periods at the end.

Reviewer: https://github.com/ansible-collections/overview/pull/229/commits/a92311160d7778b034aa3bee9c03734d10e60656
